recipes/quest/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "quest" %}
+{% set version = "3.0.a1" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/erdc/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host: 
```

Comment:
Just here should be ``host`` instead of ``build``. I already changed that, when the CI pass I will merge that
:)

Reply:
Oops, I clicked the wrong bit on GitHub, in a hurry. Yes, I meant this one :)